### ISSUE TITLE
`kin2sex` returns only '202' values for age_kin

### ISSUE THREAD

**Issue by alburezg** [IvanWilli/DemoKin#18]
Running the following code:

```
library(DemoKin)

years <- ncol(swe_px)
ages <- nrow(swe_px)
swe_surv_f_matrix <- swe_px
swe_surv_m_matrix <- swe_px ^ 1.5 # artificial perturbation for this example
swe_fert_f_matrix <- swe_asfr
swe_fert_m_matrix <- rbind(matrix(0, 5, years),  
                           swe_asfr[-((ages-4):ages),]) * 1.05 # artificial perturbation for this example

output_period <- 1950

output_kin <- c("gm", "m")

kin_out <- kin2sex(
  pf = swe_surv_f_matrix 
  , pm = swe_surv_m_matrix
  , ff = swe_fert_f_matrix 
  , fm = swe_fert_m_matrix
  , sex_focal = "f"
  , time_invariant = FALSE
  , birth_female = .5
  , output_period = output_period
  , output_kin = output_kin
)
```

returns only value '202' for age_kin:

```
table(kin_out$kin_full$age_kin)
```
  202 
  404

**Comment by IvanWilli** [comment]
Corrected in #19

**Comment by alburezg** [comment]
Works like a charm, thanks!
